### DISCUSSION TITLE
Limit output power?

### DISCUSSION THREAD

**Discussion by piepa12** [CrashOverride85/zc95#94] [Q&A]
Limit output power?

I love this project and have already built 2 devices. 
Unfortunately, my programming skills are not so good that I can program it myself. Is it possible to create an option in the menu that allows you to reduce the output power? The MK312 has an option where you can choose between medium normal and high. 
Can something like that be relegated to the zc95? 
Sometimes the output power is already way too high at 30..

I'm very happy about an answer.

Best regards 
Jonas

**Comment by CrashOverride85** [comment]
Nice, glad to hear you like the project :)

It is something I've thought about before, but never got as far as adding it. 
But you're right, it'd be a nice addition.
There's a couple of other changes I'll be making first, but I'll try and get this into the next release

**Comment by CrashOverride85** [comment]
There's now a high/medium/low selection under the output -> power level in v1.9
